src/runner.ts

```diff
@@ -215,7 +215,14 @@ export default async (options: RunnerOption): Promise<RunMigration[]> => {
     const toRun: Migration[] = getMigrationsToRun(options, runNames, migrations)
 
     if (!toRun.length) {
-      logger.info('No migrations to run!')
+      const messageToPrint = typeof options.onNoMigrationsToRunMessage === 'undefined'
+        ? 'No migrations to run!'
+        : options.onNoMigrationsToRunMessage;
```

Comment:
How about this way? 
```suggestion
      const messageToPrint = options.onNoMigrationsToRunMessage ?? 'No migrations to run!';
```

Reply:
I wasn't not sure how they transpile this library when releasing/compiling. The nullish coalescing operator has been available [only since Node 14](https://node.green/#ES2020-features--nullish-coalescing-operator-----), so this was done to ensure nothing breaks and no tests fail